agent/src/util/discovery_operator.rs

```diff
@@ -71,6 +75,7 @@ impl DiscoveryOperator {
             discovery_handler_map,
             config,
             instance_map,
+            usage_update_message_sender: Arc::new(RwLock::new(None)),
```

Comment:
Why do we need this extra channel? If we use one device plugin service, this is not needed; rather we can continue to just use the instance channel

Reply:
The change here is obsolete, please check the new code.